code/controllers/subsystem/camera.dm

```diff
@@ -0,0 +1,34 @@
+// Subsystem that controls the camera vision tiles
+SUBSYSTEM_DEF(camera)
+	name = "Camera"
+	flags = SS_BACKGROUND | SS_NO_INIT
+	priority = FIRE_PRIORITY_AMBIENCE
+	runlevels = RUNLEVEL_GAME | RUNLEVEL_POSTGAME
+	wait = 2.5 SECONDS
+	offline_implications = "AI camera view won't update. No immediate action is needed."
+	var/list/chunk_queue = list()
+
+/datum/controller/subsystem/camera/fire(resumed)
+	var/static/count = 0
+	var/list/queue = chunk_queue
```

Comment:
Im 99% sure this doesnt speed up normal `for()` on loops, only `[]` accesses.

Will need to test and look into the ASM but I think this is superfluous.

Reply:
I've changed it to not have the cached variable. The logic is sound as a for loop already does this internally